base/base_tests/stl_helpers_test.cpp

```diff
@@ -19,95 +20,117 @@ class Int
   int m_v;
 };
 
-UNIT_TEST(LessBy)
+template<template<class, class> class TContainer>
+void TestSortUnique()
```

Comment:
Why not to:
```c++
template <typename Cont>
void TestSortUnique()
{
  Cont actual = {1, 2, ... };
  my::SortUnique(actual);
  ...
}
```
And call it as:
```
TestSortUnique<vector<int>>();
...
```
It allows you to get rid of these ugly allocators.